package.json

```diff
@@ -16,22 +16,19 @@
     "build": "sh ./build.sh"
   },
   "devDependencies": {
-    "conventional-changelog-conventionalcommits": "^5.0.0",
-    "eslint": "^8.30.0",
-    "eslint-config-eslint": "^7.0.0",
-    "eslint-plugin-eslint-plugin": "^5.0.7",
+    "conventional-changelog-conventionalcommits": "^6.1.0",
+    "eslint-config-eslint": "^8.0.0",
+    "eslint-plugin-eslint-plugin": "^5.1.1",
     "eslint-plugin-node": "^11.1.0",
-    "jest": "^29.3.1",
-    "semantic-release": "^19.0.5"
-  },
-  "peerDependencies": {
-    "eslint": "^8"
+    "jest": "^29.6.1",
+    "semantic-release": "^21.0.7"
```

Comment:
> error semantic-release@21.0.7: The engine "node" is incompatible with this module. Expected version ">=18". Got "16.20.1"

That's the reason it keeps `semantic-release` to v19.

Happy to require node v18 if updating `.github/workflows/*.yaml` as well

Reply:
I updated the build to use 18.